### PR TITLE
[PVR] Guide window: Fix initialization after skin reload (trac#17546).

### PR DESCRIPTION
**Pull request by ksooo**
Fixes https://trac.kodi.tv/ticket/17546

This change has been runtime-tested on macOS, latest kodi master.

@Jalle19 for review?

### CODE REVIEW

File: xbmc/pvr/windows/GUIWindowPVRGuide.cpp [hunk 1]
```diff
@@ -74,7 +74,7 @@ void CGUIWindowPVRGuideBase::Init()
     epgGridContainer->GoToNow();
   }
 
-  if (!m_refreshTimelineItemsThread)
```
Comment (Jalle19) [inline xbmc/pvr/windows/GUIWindowPVRGuide.cpp]: So this check is no longer needed? Can the whole member be removed?

Comment (ksooo) [inline xbmc/pvr/windows/GUIWindowPVRGuide.cpp]: Yes, the check is replaced by something that not only kicks in during very first open of the window, but also after the window controls have been replaced when doing a skin reload. Much cleaner now - and fixing the bug. 

The member is used at other places in this file.

Comment (Jalle19) [inline xbmc/pvr/windows/GUIWindowPVRGuide.cpp]: +1

**Comment by xhaggi** [comment]
looks good, but was not expressly asked ;)

**Pull request merged**